aioredlock/algorithm.py

```diff
@@ -16,35 +16,19 @@ class Aioredlock:
 
     redis_connections = attr.ib(default=[{'host': 'localhost', 'port': 6379}])
 
-    lock_timeout = attr.ib(default=10.0, converter=float)
```

Comment:
Configurable `auto_extend_interval` might be useful here.

Reply:
yeah, I've changed it to be configurable